tests/test_messagedirector.py

```
@@ -0,0 +1,134 @@
+
+    def test_non_subscriber_does_not_receive(self, md, channel_conn):
+        sub = channel_conn(CH_A)
+        sub.flush()
+        sender = channel_conn()
+        sender.send(Datagram.create([CH_B], sender=0, msgtype=1234))
+        sub.expect_none(timeout=0.5)
+
+    def test_fanout_to_multiple_subscribers(self, md, channel_conn):
+        a = channel_conn(CH_A)
+        b = channel_conn(CH_A)
+        a.flush(); b.flush()
```

Ruff (0.15.12)

[error] Multiple statements on one line (semicolon)

(E702)